nginx.master.conf

```diff
@@ -266,47 +266,48 @@ http {
             proxy_redirect http://$http_host/ /pkgpanda/;
         }
 
-        location /system/logs/v1/ {
+        location /system/v1/logs/v1/ {
             access_by_lua 'auth.validate_jwt_or_exit()';
-            proxy_buffering off;
-            proxy_cache off;
-
-            # make event source work with nginx
-            # http://stackoverflow.com/questions/13672743/eventsource-server-sent-events-through-nginx
-            proxy_set_header Connection '';
-            proxy_http_version 1.1;
-            chunked_transfer_encoding off;
+            include common/server-sent-events.conf;
 
             proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
             proxy_set_header Host $host;
             proxy_set_header X-Forwarded-Proto $scheme;
             proxy_pass http://log/;
         }
 
-        location ~ ^/system/logs/v1/(slave|agent)/(?<agentid>[0-9a-zA-Z-]+)(?<url>.*)$ {
+        location ~ ^/system/v1/agent/(?<agentid>[0-9a-zA-Z-]+)/logs/v1/(?<url>.*)$ {
             access_by_lua 'auth.validate_jwt_or_exit()';
             set $agentaddr '';
             more_clear_input_headers Accept-Encoding;
             rewrite ^/(slave|agent)/[0-9a-zA-Z-]+/.*$ $url break;
             rewrite_by_lua_file conf/master/agent.lua;
 
-            proxy_buffering off;
-            proxy_cache off;
+            include common/server-sent-events.conf;
 
-            # make nginx with with server sent events
-            # http://stackoverflow.com/questions/13672743/eventsource-server-sent-events-through-nginx
-            proxy_set_header Connection '';
-            proxy_http_version 1.1;
-            chunked_transfer_encoding off;
+            proxy_set_header Host $http_host;
```

Comment:
would be nice to pull the proxy_set_header into an include file if we can rather than duplicating it.